aep/general/0231/aep.md.j2

```diff
@@ -9,25 +9,40 @@ provides this functionality.
 APIs **may** support batch get to retrieve a consistent set of resources.
 
 - The method's name **must** begin with `BatchGet`. The remainder of the method
-  name **should** be the plural form of the resource being retrieved.
+  name **must** be the plural form of the resource being retrieved.
 - The HTTP verb **must** be `GET`.
-- The HTTP URI **must** end with `:batchGet`.
-- The URI path **should** represent the collection for the resource, matching
-  the collection used for simple CRUD operations. If the operation spans
-  parents, a dash (`-`) **may** be accepted as a wildcard.
+- The HTTP URI **must** end with `:BatchGet`.
```

Comment:
Why the casing change here?  aep.dev/custom-methods still uses camelCase rather than PascalCase.